src/carriers/human_carrier/HumanStream.cpp

```diff
@@ -35,6 +35,6 @@ void HumanStream::write(const Bytes& b) {
         show.append(outputCache.c_str(),idx);
         std::cout << "*** TYPE THIS ON THE OTHER TERMINAL: " << show << std::endl;
         outputCache = outputCache.substr(idx+1);
-        Time::delay(1);
+        SystemClock::delaySystem(1);
```

Comment:
In general, I'd be more in favour of avoiding some method calls and use C++11 methods:
* `std::this_thread::sleep_for()` instead of `delaySystem()`
* `std::chrono::system_clock::now()` instead of `nowSystem()`